src/Symfony/Component/Notifier/Bridge/Telegram/composer.json

```diff
@@ -17,8 +17,8 @@
     ],
     "require": {
         "php": ">=8.1",
-        "symfony/http-client": "^5.4|^6.0|^7.0",
-        "symfony/notifier": "^6.2.7|^7.0"
+        "symfony/http-client": "^6.3|^7.0",
```

Comment:
why this bump?

Reply:
we need #49911

Reply:
and for this we need `symfony/mime` also